core/Updates/4.0.0-b1.php

```diff
@@ -48,11 +50,35 @@ public function getMigrations(Updater $updater)
             $migrations[] = $this->migration->plugin->activate('CustomJsTracker');
         }
 
+        if ($this->usesGeoIpLegacyLocationProvider()) {
+            // activate GeoIp2 plugin for users still using GeoIp2 Legacy (others might have it disabled on purpose)
+            $migrations[] = $this->migration->plugin->activate('GeoIp2');
+        }
+
+        // remove old options
+        $migrations[] = $this->migration->db->sql('DELETE FROM `' . Common::prefixTable('option') . '` WHERE option_name IN ("geoip.updater_period", "geoip.loc_db_url", "geoip.isp_db_url", "geoip.org_db_url")');
+
```

Comment:
Should there be a migration that deletes the old GeoIp plugin?

Reply:
There was no `GeoIp` plugin. That stuff was included in the `UserCountry` plugin, which still exist

Reply:
👍 missed that